website/views.py

```diff
@@ -27,7 +27,57 @@ def dashboard():
 
 @views.route("/athleteDashboard")
 def athleteDashboard():
-    return render_template("athleteDashboard.html", user=current_user)
+
+    dfR = pd.read_csv('website/data/readiness.csv')

```

Comment:
This should be fine for the current milestone. I think the longer term goal is to read this info in from the database, as we want it to be the permanent source of truth for our data, rather than csv's on the website's filesystem.